integrations/appboy/lib/index.js

```diff
@@ -110,50 +105,18 @@ Appboy.prototype.initializeV1 = function(customEndpoint) {
   this._shim = window.appboy.initialize;
 
   this.load('v1', function() {
-    var config = {};
-    var datacenterMappings = {
-      us: 'https://sdk.iad-01.braze.com',
-      us02: 'https://sdk.iad-02.braze.com',
-      us03: 'https://sdk.iad-03.braze.com',
-      eu: 'https://sdk.fra-01.braze.eu'
-    };
-    if (options.safariWebsitePushId)
-      config.safariWebsitePushId = options.safariWebsitePushId;
-    if (options.enableHtmlInAppMessages) config.enableHtmlInAppMessages = true;
-
-    // Setup custom endpoints
-    if (options.customEndpoint) {
-      var endpoint = options.customEndpoint;
-      var regex = new RegExp('^(http|https)://', 'i');
-      config.baseUrl =
-        (regex.test(endpoint) ? endpoint : 'https://' + endpoint) + '/api/v3';
-    } else {
-      config.baseUrl =
-        (datacenterMappings[options.datacenter] ||
-          'https://sdk.iad-01.braze.com') + '/api/v3';
+    if (appboyUtil.shouldOpenSession(userId, options)) {
+      self.hasBeenInitialized = true;
```

Comment:
New `hasBeenInitialized` state variable is used to conditionally block requests until context userId is `identify`-ed.